aqi_monitor/aqi_monitor/settings.py

```diff
@@ -104,7 +104,7 @@
 # Internationalization
 # https://docs.djangoproject.com/en/3.0/topics/i18n/
 
-LANGUAGE_CODE = "en-us"
+LANGUAGE_CODE = "ru-ru"
```

Comment:
I'm not sure that we have to proceed with Russian =)

Reply:
Fixed

Reply:
Doesn't looks like fixed =)